web/docs/understand/query-language/operators/drop.md

```diff
@@ -6,13 +6,32 @@ entire schemas.
 The `drop` operator is the dual to [`select`](select), which selects a given set
 of fields from the output.
 
-## Parameters
+## Synopsis
 
-- `fields: [string]`: The extractors of fields to drop.
-- `schemas: [string]`: The names of schemas to drop.
+```
+drop EXTRACTORS[, …]
+```
+
+### Extractors
+
+The extractors of the fields or schemas to drop.
 
 ## Example
 
+Drop the fields `source_ip` and `dest_ip`, and all schemas of type
+`suricata.dns`:
+
+```
+drop source_ip, dest_ip, :suricata.dns
+```
+
+## YAML Syntax Example
+
+:::info Deprecated
+The YAML syntax is deprecated, and will be removed in a future release. Please
+use the pipeline syntax instead.
+:::
```

Comment:
```suggestion
:::info Deprecated
The YAML syntax is deprecated since version 3.0, and will be removed in a future release. Please
use the pipeline syntax instead.
:::
```
Is it feasible to pin down the actual version where this has been declared deprecated? It would give a nice hint for users who have to update their `version<3` VASTs, especially after a longer amount of time has passed.

Reply:
Done in 6e3fd17e2 for all operators.